web/src/components/svgs/AIRtecAwhite.js

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
```

Comment:
I don't mind `import *` instead of `import React` but let's pick one style and stay consistent.

Reply:
Got it @asasmith